### ISSUE TITLE
Support hybrid cars

### ISSUE THREAD

**Issue by domoritz** [WJDDesigns/Ultra-Vehicle-Card#3]
The card is pretty neat. It would be awesome to support hybrid cars that have both electric and gas range.

**Comment by WJDDesigns** [comment]
This will probably show up in the next update as this is a top need.

**Comment by WJDDesigns** [comment]
<img width="482" alt="Screenshot 2024-08-04 at 12 17 26 PM" src="https://github.com/user-attachments/assets/f485fbdf-4eab-4d0c-933d-10c74e949feb">
Coming soon!

**Comment by AlexandreLogist** [comment]
I would say I prefer to have the fuel first, as this is the primary for a hybrid car? 
Or will it be configurable ? Thank you !

**Comment by WJDDesigns** [comment]
Yeah, I almost did it that way so I'll change it to that. Thanks!

**Comment by domoritz** [comment]
Good to be configurable. For PHEVs electric can be the primary source (it certainly is for me).

**Comment by WJDDesigns** [comment]
Added in [V1.1.4](https://github.com/WJDDesigns/Ultra-Vehicle-Card/releases/tag/1.1.4) Thanks for the suggestion!

**Comment by brunomdteles** [comment]
I cannot't add a hybrid option, version 1.2.3
